core/kernel/asan.c

```diff
@@ -275,9 +275,8 @@ void __noreturn __asan_report_store_n_noabort(vaddr_t addr, size_t size)
 }
 
 void __asan_handle_no_return(void);
-void __noreturn __asan_handle_no_return(void)
+void __asan_handle_no_return(void)
 {
-	panic();
```

Comment:
no kind of panicking ?

Reply:
No, it's not entirely clear to me what we're supposed to do here. The Linux kernel does nothing though.